pkg/sensors/program/loader_windows.go

```diff
@@ -0,0 +1,322 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package program
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/link"
+	"github.com/cilium/tetragon/pkg/bpf"
+	"github.com/cilium/tetragon/pkg/constants"
+	"github.com/cilium/tetragon/pkg/logger"
+	"github.com/cilium/tetragon/pkg/sensors/unloader"
+	"golang.org/x/sys/windows"
+)
+
+var (
+	notSupportedWinErr     = errors.New("not supported on windows")
+	programTypeProcessGUID = makeGUID(0x22ea7b37, 0x1043, 0x4d0d, [8]byte{0xb6, 0x0d, 0xca, 0xfa, 0x1c, 0x7b, 0x63, 0x8e})
+	attachTypeProcessGUID  = makeGUID(0x66e20687, 0x9805, 0x4458, [8]byte{0xa0, 0xdb, 0x38, 0xe2, 0x20, 0xd3, 0x16, 0x85})
+)
+
+func makeGUID(data1 uint32, data2 uint16, data3 uint16, data4 [8]byte) windows.GUID {
+	return windows.GUID{Data1: data1, Data2: data2, Data3: data3, Data4: data4}
+}
+
+func winAttachStub(_ *ebpf.Collection, _ *ebpf.CollectionSpec,
+	prog *ebpf.Program, spec *ebpf.ProgramSpec) (unloader.Unloader, error) {
+
+	return nil, notSupportedWinErr
+}
+
+func RawAttachWithFlags(targetFD int, flags uint32) AttachFunc {
+	return winAttachStub
+}
+
+func TracepointAttach(load *Program, bpfDir string) AttachFunc {
```

Comment:
could we get away with removing this one and others, like suggested in here https://github.com/cilium/tetragon/commit/16b67c77bf4b7d265e930504377e2f29297deb53

I think we don't need those attach functions in windows code,
but I can't check and it's possible I'm missing something

it might be helpful to have target that would build windows go code on linux,
just to check if we broken something with our change

Reply:
We need these as stub functions for compilation.

Reply:
Actually I can remove some but not all stub functions. Let me change..

Reply:
I have removed some more unused functions